### PR TITLE
Fixed issue #5001 | JavaScript program to print inverted pyramid pattern 

### LINKED ISSUE #5001: Write a JavaScript program to print inverted pyramid pattern
### Description

Write a JavaScript program to print inverted pyramid pattern

```
Input   : 5

Output : 
1 2 3 4 5
1 2 3 4 
1 2 3
1 2
1
```

<sub>Contributed by - @PraaneshSelvaraj</sub>
<details>
<summary>How to contribute</summary>

- Save the solution in `PrintInvertedPyramidPattern.js` file
- Add `PrintInvertedPyramidPattern.js` file in `program/print-inverted-pyramid-pattern` folder

</details>

### PR DESCRIPTION
**Pull request by SarthakSanjay**
### Why:

Closes codinasion/codinasion#3660 

### What's being changed:
Added JavaScript program to print inverted pyramid pattern

### CODE REVIEW

**Review by oyetanishq — DISMISSED** [review]

File: program/print-inverted-pyramid-pattern/PrintInvertedPyramidPattern.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//inverted pyramid
+for(let a = 6;a>=1;a--){
+    let str =' '
```
Comment (anandfresh) [inline program/print-inverted-pyramid-pattern/PrintInvertedPyramidPattern.js:3]: This code gives an extra double quotes in the last line .
To fix this make the for loop condition as below
`for(let a = 6;a>1;a--){`

**Review by anandfresh — CHANGES_REQUESTED** [review]

File: program/print-inverted-pyramid-pattern/PrintInvertedPyramidPattern.js (hunk 1 above)
Comment (SarthakSanjay) [reply program/print-inverted-pyramid-pattern/PrintInvertedPyramidPattern.js:3]: Ok

File: program/print-inverted-pyramid-pattern/PrintInvertedPyramidPattern.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    for(let i = 1;i<=a-1;i++){
+        str+=i + " "
```
Comment (anandfresh) [inline program/print-inverted-pyramid-pattern/PrintInvertedPyramidPattern.js:5]: You should not give 6 and print for 5 pyramids .
It has to print for the given number. 
I have fixed your code to work as expected.

```
for(let a = 5;a>=1;a--){
    let str =' '
    for(let i = 1;i<=a;i++){
        str+=i + " "
    }
    console.log(str)
}
```

**Review by anandfresh — CHANGES_REQUESTED** [review]

**Review by anandfresh — APPROVED** [review]

**Review by harshraj8843 — APPROVED** [review]
Thanks for contributing 👍🏻

**Pull request merged**